lib/handler/message_create.go

```diff
@@ -14,57 +11,32 @@ import (
 )
 
 func MessageCreate(session *discordgo.Session, orgMsg *discordgo.MessageCreate) {
-	defer func() {
-		err := recover()
-		if err != nil {
-			log.Print("Oops, ", err)
-			debug.PrintStack()
-		}
-	}()
+	guild := db.LoadGuild(&orgMsg.GuildID)
+	prefix := guild.Prefix
 
-	if config.CurrentConfig.Debug {
-		start := time.Now()
-		defer func() {
-			log.Print("Message processed in ", time.Since(start).Milliseconds(), "ms.")
-		}()
+	if orgMsg.Author.ID == session.State.User.ID {
+		return
 	}
 
-	guild := db.LoadGuild(&orgMsg.GuildID)
-
-	// Ignore all messages created by the bot itself
-	// This isn't required in this specific example but it's a good practice.
-	if orgMsg.Author.ID == session.State.User.ID || orgMsg.Content == "" {
+	if orgMsg.Author.Bot {
 		return
 	}
 
-	// Ignore all messages from blacklisted user
 	for _, v := range config.CurrentConfig.UserBlacklist {
 		if orgMsg.Author.ID == v {
 			return
 		}
 	}
 
-	isCmd := false
-	var trimedMsg string
-	if strings.HasPrefix(orgMsg.Content, guild.Prefix) {
-		isCmd = true
-		trimedMsg = strings.TrimPrefix(orgMsg.Content, guild.Prefix)
-	} else if strings.HasPrefix(orgMsg.Content, session.State.User.Mention()) {
-		isCmd = true
-		trimedMsg = strings.TrimPrefix(orgMsg.Content, session.State.User.Mention())
-		trimedMsg = strings.TrimPrefix(trimedMsg, " ")
-	}
-	if isCmd {
-		if config.CurrentConfig.Debug {
-			log.Print("Command processing")
-		}
-		cmds.HandleCmd(session, orgMsg, guild, &trimedMsg)
+	if !strings.HasPrefix(orgMsg.Content, prefix) {
+		urlCheck(session, orgMsg)
```

Comment:
I just don't want urlCheck() to run when the command is used, so there is no problem.

Reply:
I don't see the need to be able to use commands when mentioned.

Reply:
> I don't see the need to be able to use commands when mentioned.
It is copy from Vanilla.
It was the way to access command when you forget the prefix.
It can be removed if prefix is not going to be configurable with command.

Reply:
Oh! Sry, I'll fix them.